src/screens/SignUp/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
```

Comment:
Why use this? you can use only react-hook-form, I don't like the idea to add more library when are unnecessary